eclipse-scout-core/src/carousel/Carousel.ts

```diff
@@ -58,7 +58,9 @@ export class Carousel extends Widget implements CarouselModel {
 
   protected override _render() {
     // add container
-    this.$container = this.$parent.appendDiv('carousel');
+    this.$container = this.$parent.appendDiv('carousel')
+      // Tooltips on elements in the filmstrip will check all scroll parents (i.e. elements marked with 'scrollable') if the anchor is visible.
```

Comment:
Is this comment still accurate? We are taking advantage of the fact that the tooltip NOT only considers scroll parents with 'scrollable' classes but also pseudo scroll parents.